sql/driver/conn.go

```diff
@@ -52,6 +54,10 @@ func (c *conn) Exec(stmt string, args []driver.Value) (driver.Result, error) {
 }
 
 func (c *conn) Query(stmt string, args []driver.Value) (*rows, error) {
+	if c.beginTransaction {
+		stmt = "BEGIN TRANSACTION;" + stmt
```

Comment:
While it isn't strictly necessary, I think it will look better to place a space after the semicolon if we ever print the statement on the server side.